src/app/shared/modal/modal-login/modal-login.component.ts

```diff
@@ -17,8 +17,13 @@ export class ModalLoginComponent implements OnInit {
   password!: string;
   isButtonEnabled: boolean = false;
   id: number = 1;
+  token?: ''; 
```

Comment:
It is better to write the type directly like **string**

Reply:
ok